src/main/java/org/pethotel/HeavenForPets/entity/PetEntity.java

```diff
@@ -14,29 +13,34 @@
 public class PetEntity {
     @Id
```

Comment:
Please, in this class space between fields, one empty line after filed, for readability.